index.bs

```diff
@@ -521,29 +521,19 @@ which would have issues of its own.
 The term <dfn id="concept-device-sensor">device sensor</dfn> refers to a device's underlying
 physical sensor instance.
 
-The [=device sensors=] measure different physical quantities
-and provide corresponding <dfn>raw sensor readings</dfn>
-which are a source of information about the user and their environment.
+A [=device sensor=] measures different physical quantities
+and provides a corresponding <dfn>sensor reading</dfn>
+which is a source of information about the environment.
 
-Each [=raw sensor reading|reading=] is composed of the <dfn lt="reading value">values</dfn>
-of the different physical quantities measured by the [=device sensor|sensor=]
+Each [=sensor reading=] is composed of the <dfn lt="reading value">values</dfn>
+of the different physical quantities measured by the [=device sensor=]
```

Comment:
Same as above, unclear why to use `different physical quantities` rather than `physical quantity`.

Reply:
Fixed.